bin/cliain/src/validators.rs

```diff
@@ -0,0 +1,17 @@
+use log::info;
+use aleph_client::{change_members, create_connection, KeyPair};
+use substrate_api_client::{AccountId, XtStatus};
+use sp_core::crypto::Ss58Codec;
+
+/// Change validators to the provided list by calling the provided node.
+/// The keypair has to be capable of signing sudo calls.
+pub fn change(validators: Vec<String>, node: String, key: KeyPair) {
```

Comment:
We can change the name `key` to `sudo_key` to reflect this function usage better.